libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/Indexed.hs

```diff
@@ -307,4 +314,4 @@ instance PrettyA (SKey kr c) where
 instance PrettyA (MultiAsset era) where
   prettyA (MultiAsset v) = prettyA v
 
-type Good c = DSignable c (CH.Hash (CC.HASH c) EraIndependentTxBody)
+type Goode c = DSignable c (CH.Hash (CC.HASH c) EraIndependentTxBody)
```

Comment:
goodie goodie gumdrops?

Reply:
This was a leftover wart. It has been removed.